CONTRIBUTING.md

```diff
@@ -197,5 +197,22 @@ to merge**.
   - Send staging tag as PR for review.
 - Create a tag `v{version}` on the merged PR and push remote.
 
+## Merging existing ECS conventions
+
+The Elastic Common Schema (ECS) is being merged into OpenTelemetry Semantic
+Conventions per [OTEP 222][otep222]. When adding a semantic convention that
+exists in some form in ECS, consider the following guidelines:
+
+- Prefer using the existing ECS name when possible. In particular:
+  - Do not modify an ECS name for the only reason of complying with
+    [OpenTelemetry general attribute naming guidelines](https://opentelemetry.io/docs/specs/otel/common/attribute-naming/).
```

Comment:
We discussed this PR in the working group meeting on Nov 27. The general feeling is that this bullet point is "too permissive" and pretty much opens up to disregard completely the guidelines in place, including the ones which are important (e.g. character set).

We discussed these options/alternatives/improvements

1. Adapt this bullet point to better indicate which guidelines shouldn't be applied, or adapt the text such as it is not so broad 
2. Remove this paragraph completely
3. Close this PR and deal with problematic/conflicting ECS attributes as they appear
4. Indicate here that new attributes should be checked on ECS first if they exist there before suggesting new attributes in OTel to avoid even more conflicts

The conclusion was we should aim to do a mix of 1, and 4

Reply:
Great summary @joaopgrassi 

I think logic-wise it's better start with second bullet point 
```
-  If proposing a name that differs from the ECS convention, provide usage
    data, user issue reports, feature requests, examples of prior work on a
    different standard or comparable evidence about the alternatives.
```
And then add more clarification, because most of the problems coming from `Name Pluralization guidelines`, everything else from naming guidelines are the same for ECS as well
```
- When no suitable alternatives are provided, altering an ECS name solely 
  for the purpose of complying with [Name Pluralization guidelines] (https://opentelemetry.io/docs/specs/otel/common/attribute-naming/#name-pluralization-guidelines) 
  SHOULD BE avoided.
- Before proposing a new attribute, it's advisable to check ECS first to ensure 
  a similar attribute doesn't already exist. If a match is found, consider adopting it 
  into Otel to minimize potential conflicts.
```

Reply:
i'd suggest slightly more permissive still: `SHOULD BE avoided` -> `MAY BE avoided`

Reply:
Resolving per approval from Joao :)